src/arrayMethodSort.js

```diff
@@ -4,9 +4,37 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareAsStrings) {
+    let counter;
+
+    do {
+      counter = 0;
+
+      for (let i = 1; i < this.length; i++) {
+        const prevValue = this[i - 1];
+        const currentValue = this[i];
+
+        if (compareFunction(prevValue, currentValue) > 0) {
+          counter++;
+          this[i - 1] = currentValue;
+          this[i] = prevValue;
+        }
+      }
+    } while (counter > 0);
+
+    return this;
   };
 }
 
+const compareAsStrings = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  }
```

Comment:
`else` is redundant, but condition still needs to be checked.
```suggestion
  if (stringA > stringB) {
    return 1;
  }
  
  if (stringA === stringB) {
    return 0;
  }
```